sonar-check-api/src/main/java/org/sonar/check/BelongsToProfile.java

```diff
@@ -26,13 +26,15 @@
 
 /**
  * @since 2.1
+ * @deprecated since 5.6 use your own way to implement org.sonar.api.profiles.ProfileDefinition 
  */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface BelongsToProfile {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)